carbonmark-api/src/routes/users/get.schema.ts

```diff
@@ -14,11 +15,11 @@ export const Params = Type.Object(
 export const QueryString = Type.Object({
   type: Type.Optional(
     Type.Union([Type.Literal("wallet"), Type.Literal("handle")], {
-      description:
-        "When providing an wallet `0x` address instead of a handle, you must attach the `type=wallet` query parameter",
+      description: "Deprecated.",
```

Comment:
Since we're versioned (theoretically) I'd be tempted to nuke this rather than leave it hanging around